ui/src/app/pages/access/aar-title.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import '@testing-library/jest-dom';
+
+import * as React from 'react';
+
+import { AccessModule, ConfigResponse, ProfileApi } from 'generated/fetch';
+
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { AARTitle } from 'app/pages/access/aar-title';
+import {
+  queryForCTTitle,
+  queryForRTTitle,
+} from 'app/pages/access/compliance-training-module-card-title.spec';
+import { DARTitleProps } from 'app/pages/access/dar-title';
+import { createEmptyProfile } from 'app/pages/login/sign-in';
+import { registerApiClient } from 'app/services/swagger-fetch-clients';
+import { serverConfigStore } from 'app/utils/stores';
+
+import defaultServerConfig from 'testing/default-server-config';
+import { ProfileApiStub } from 'testing/stubs/profile-api-stub';
+
+const createProps = (): DARTitleProps => ({
+  moduleName: AccessModule.COMPLIANCE_TRAINING,
+  profile: {
+    ...createEmptyProfile(),
+  },
+});
+
+const setup = (
+  props = createProps(),
+  config: ConfigResponse = { ...defaultServerConfig }
+) => {
+  registerApiClient(ProfileApi, new ProfileApiStub());
+  serverConfigStore.set({ config });
+  render(<AARTitle {...props} />);
+  return userEvent.setup();
+};
+
+describe(AARTitle.name, () => {
```

Comment:
This doesn't completely cover the newly-extracted components. I think that's OK since this is an improvement over the complete lack of test coverage we had before and keeps the PR in a reasonable scope.